action/action-router.js

```diff
@@ -0,0 +1,36 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3'
+    }
+}
+
+const db = knex(knexConfig);
+
+
+router.get('/', (req, res) => {
+    db('actions')
+    .then(actions => {
+        res.status(200).json(actions)
+    })
+    .catch(err => {
+        res.status(500).json(err)
+    })
+});
+
+router.post('/', async (req, res) => {
```

Comment:
Stick to .then() or try/catch for consistency. Also I don't see a relationship of an action to a project. Ideally we would want to be posting an action to a url of 'http://localhost:XXX/api/projects/:id/action` where you can take the id from the parameters and assign it to a `project_id` field in your projects table as a foreign key.